library/Freckle/App/Kafka/Consumer.hs

```diff
@@ -0,0 +1,166 @@
+{-# LANGUAGE ApplicativeDo #-}
+
+module Freckle.App.Kafka.Consumer
```

Comment:
I wonder if we need to rename `Freckle.App.Kafka` to `Freckle.App.Kafka.Producer`?

Reply:
Sounds right, and then `Freckle.App.Kafka` could re-export both.